docker/docker-entrypoint.sh

```diff
@@ -8,8 +8,12 @@ set -e
 DB_WAIT_TIMEOUT=${DB_WAIT_TIMEOUT-3}
 MAX_DB_WAIT_TIME=${MAX_DB_WAIT_TIME-30}
 CUR_DB_WAIT_TIME=0
+
+# set NAUTOBOT_DOCKER_SKIP_INIT to NAUTOBOT_DOCKER_SKIP_INIT, defaulting to false:
 NAUTOBOT_DOCKER_SKIP_INIT=${NAUTOBOT_DOCKER_SKIP_INIT-false}
-if [ "$NAUTOBOT_DOCKER_SKIP_INIT" == "false" ]; then
+# lowercase NAUTOBOT_DOCKER_SKIP_INIT:
+NAUTOBOT_DOCKER_SKIP_INIT=$(echo $NAUTOBOT_DOCKER_SKIP_INIT | tr '[:upper:]' '[:lower:]')
+if [[ "$NAUTOBOT_DOCKER_SKIP_INIT" == "false" ]]; then
```

Comment:
Is this change (`[]` to `[[]]`) what fixed the issue reported in #1227:

> Our testing shows even setting it to "false" does not work and migrations init is skipped.

?

Reply:
I *think* so, although I tested with just single square brackets and it seemed to work. I think double square brackets is the way to go though. Following taken from a quick Google:

```[] is for math orerations 
[[ ]] for pattern matching, 


eg 1 ###########
eg: 
if [ $marks -ge 80 ];
then
	do something
fi```